src/core/markovProcessGraph.js

```diff
@@ -347,15 +347,11 @@ export class MarkovProcessGraph {
       return result;
     })();
 
-    const timeBoundaries = (() => {
-      const edgeTimestamps = Array.from(
-        wg.graph.edges({showDangling: false})
-      ).map((x) => x.timestampMs);
-      const start = min(edgeTimestamps);
-      const end = max(edgeTimestamps);
-      const boundaries = weekIntervals(start, end).map((x) => x.startTimeMs);
-      return [-Infinity, ...boundaries, Infinity];
-    })();
+    const timeBoundaries = [
+      -Infinity,
+      ...intervals.map((x) => x.startTimeMs),
```

Comment:
We depend on `intervals` being sorted; could you please check that it
is? (Probably want `timeBoundaries` to be strictly-sorted, in fact.)

Reply:
Done via #2323